### PR TITLE
DOC Bibtex fix for spatial_resolution.py

### LINKED ISSUE #8925: use bibtex in spatial_resolution.py
convert references in `mne/minimum_norm/spatial_resolution.py` to use footcite / footbibliography

### PR DESCRIPTION
**Pull request by NeuroLaunch**
#### Reference issue
Example: Fixes #8925.

#### What does this implement/fix?
Adds footcite and footbibliography support for `resolution_metrics()` in _minimum_norm/spatial_resolution.py_.

#### Collaborator
@cmista

### CODE REVIEW

File: mne/minimum_norm/spatial_resolution.py [hunk 1]
```diff
@@ -65,19 +65,14 @@ def resolution_metrics(resmat, src, function='psf', metric='peak_err',
 
     Notes
     -----
-    For details, see [1]_ [2]_.
+    For details, see :footcite:`MolinsEtAl2008` :footcite:`HaukEtAl2019`.
```
Comment (sappelhoff) [inline mne/minimum_norm/spatial_resolution.py:68]: the footcite directive is documented here: https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#role-footcite

I think you could do something like this here:

```suggestion
    For details, see `:footcite:`MolinsEtAl2008,HaukEtAl2019`.
```

File: mne/minimum_norm/spatial_resolution.py [hunk 2]
```diff
@@ -65,19 +65,14 @@ def resolution_metrics(resmat, src, function='psf', metric='peak_err',
...
 
     .. versionadded:: 0.20
-
+
```
Comment (sappelhoff) [inline mne/minimum_norm/spatial_resolution.py:71]: you added some trailing whitespace here: That's undesirable (invisible, useless characters) --> a good solution would be to look into your editor settings. For most editors you can have an automatic removal of trailing whitespace :-)

File: mne/minimum_norm/spatial_resolution.py [hunk 3]
```diff
@@ -332,3 +327,8 @@ def _rectify_resolution_matrix(resmat):
                     (shape[0], shape[1], resmat.shape[0], resmat.shape[1]))
 
     return resmat
+
+###############################################################################
+# References
+# ----------
+# .. footbibliography::
```
Comment (sappelhoff) [inline mne/minimum_norm/spatial_resolution.py:334]: why is this necessary?

Comment (drammock) [reply mne/minimum_norm/spatial_resolution.py:334]: agree, this is not needed.  in a *code* file the references section goes within the docstring where the reference is cited.  In a *tutorial* file you want something like this just once at the very end.

Comment (NeuroLaunch) [reply mne/minimum_norm/spatial_resolution.py:334]: That was a remnant, so we'll get that out.

File: mne/minimum_norm/spatial_resolution.py (hunk 1 above)
Comment (NeuroLaunch) [reply mne/minimum_norm/spatial_resolution.py:68]: Just saw this, we'll update it on next commit.

**Review by drammock — APPROVED** [review]

**Review by rob-luke — CHANGES_REQUESTED** [review]
I think two references were missed at 

https://github.com/mne-tools/mne-python/blob/dbdf70f280bf7ae93a61b0c3248bcb7d15aba0a4/mne/minimum_norm/spatial_resolution.py#L47

**Comment by NeuroLaunch** [comment]
> 
> 
> I think two references were missed at
> 
> https://github.com/mne-tools/mne-python/blob/dbdf70f280bf7ae93a61b0c3248bcb7d15aba0a4/mne/minimum_norm/spatial_resolution.py#L47

Indeed they were (we text searched, but not for dual citations like that).  I'll fix that now and re-push.

**Review by rob-luke — APPROVED** [review]

**Comment by sappelhoff** [comment]
@NeuroLaunch your commits look like your `.gitconfig` is not configured :thinking: 

Can you take a look at `~/.gitconfig` (if you are on Mac or Linux) on Windows, this is probably `C:\Users\username\.gitconfig`

The first lines in the file should look somewhat like this:

```
[user]
        email = stefan.appelhoff@mailbox.org
        name = Stefan Appelhoff
```

If you use an email that you also used to register in GitHub, the GitHub User Interface will be better able to link your contributions to your account.

**Comment by NeuroLaunch** [comment]
Good catch, @sappelhoff !  I had a typo.  It explains some odd behavior with my authentication (despite my ssh setup).
